libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -18,6 +18,7 @@
 #include <SRV_Channel/SRV_Channel.h>
 #include <GCS_MAVLink/GCS.h>
 #include <AP_BoardConfig/AP_BoardConfig.h>
+#include <AP_RangeFinder/RangeFinder_Backend.h>
```

Comment:
We don't usually include other headers then the main library one. I'm actually in favor of doing it (include only what you need), but not sure what direction team wants to go.

In any case, this specific one seems wrong, you are calling a method in the Rangefinder class, but not including it.